[BUG]: test server startup detection too early for GitHub

### Describe the issue:

Startup of the test server is detected by a scan on the server logging output to stdout. Startup is triggered on the beginning of the log output. This is too early as on GitHub it takes longer than on a home machine until the server accepts connects.

Startup detection should scan on for the proper log output.

### Reproduceable code example:

```python
N/A
```


### Error message:

```shell
tests/test_server.py::test_server /home/runner/work/EOS/EOS/.pytest_cache/d/.xprocess/akkudoktoreosserver$ /opt/hostedtoolcache/Python/3.12.7/x64/bin/python -m akkudoktoreosserver.flask_server
process 'akkudoktoreosserver' started pid=2229
2024-03-28 02:00:00+01:00

 * Serving Flask app 'flask_server'
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/coverage/inorout.py:505: CoverageWarning: Module modules was never imported. (module-not-imported)

akkudoktoreosserver process startup detected
  self.warn(f"Module {pkg} was never imported.", slug="module-not-imported")
FAILED

=================================== FAILURES ===================================
_________________________________ test_server __________________________________

self = <urllib3.connection.HTTPConnection object at 0x7ff3fd477aa0>

    def _new_conn(self) -> socket.socket:
        """Establish a socket connection and set nodelay settings on it.
    
        :return: New socket connection.
        """
        try:
>           sock = connection.create_connection(
                (self._dns_host, self.port),
                self.timeout,
                source_address=self.source_address,
                socket_options=self.socket_options,
            )

/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/urllib3/connection.py:199: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/urllib3/util/connection.py:85: in create_connection
    raise err
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

address = ('127.0.0.1', 8503), timeout = None, source_address = None
```


### Version information:

a41aea824c1ee19da361993c9e5867fa67dd2e99

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -31,13 +31,13 @@ def server(xprocess):
         args = [sys.executable, "-m", "akkudoktoreosserver.flask_server"]
 
         # startup pattern
-        pattern = "Serving Flask app 'flask_server'"
+        pattern = "Debugger PIN:"
         # search the first 12 lines for the startup pattern, if not found
         # a RuntimeError will be raised informing the user
         max_read_lines = 12
 
         # will wait for 10 seconds before timing out
-        timeout = 10
+        timeout = 30
 
         # xprocess will now attempt to clean up upon interruptions
         terminate_on_interrupt = True

